src/rdkafka.h

```diff
@@ -9850,6 +9906,131 @@ RD_EXPORT void rd_kafka_DeleteAcls(rd_kafka_t *rk,
 
 /**@}*/
 
+/**
+ * @name Admin API - Elect Leaders
+ * @{
+ *
+ *
+ *
+ */
+
+/**
+ * @brief Start leader election for the specified Topic Partitions.
+ */
```

Comment:
Check other places for the description for this kind of struct and change the description.